doc/tutorials/error_analysis/error_analysis_part1.ipynb

```diff
@@ -0,0 +1,459 @@
+    "A confidence interval (CI) specifies a range of numbers within which the unknown true mean value lies with a certain probability. A common confidence interval (CI) is the $95~\\%$ CI, which would contain the true value with probability $95~\\%$. Care must be taken interpreting the CI, since the lower and upper bound of a CI are themselves random variables. Just as a simulation run drafts samples from the overall ensemble, determining a CI from a simulation run is drafting a CI from all possible CIs. When the upper and lower bound of a CI have been calculated, this range either contains the true value or not, so there no longer is a probability attached to it. However, for repeated simulations with subsequent computation of the corresponding CIs, on average $95~\\%$ of CIs will contain the true value, while $5~\\%$ won't.\n",
+    "\n",
+    "If the samples are normally distributed and the SEM is known, the upper and lower bound of the $95~\\%$ CI are $\\overline{X}\\pm1.96s$.\n",
+    "\n",
+    "### Interquartile range\n",
+    "The interquartile range denotes the range, within which the central $50~\\%$ of all samples lie, if one were to order them by their size. This leaves one quarter of all samples lying below the interquartile range, and another quarter of all samples above it.\n",
+    "\n",
+    "### Now &ndash; what do we use?\n",
+    "We are interested in the precision of our overall, averaged, simulation result, and not in the precision of the individual samples. Those are expected to fluctuate, and in many cases, those fluctuations are uninteresting for the end result. Out of the options presented above, the SEM and the CI are the only ones doing this requirement justice. Since they are related, the question boils down to how to compute the SEM, which will be the topic of the rest of this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Uncorrelated samples\n",
+    "How the SEM can be computed depends on the data itself. For uncorrelated samples, it is nearly trivial:\n",
+    "\n",
+    "$\n",
+    "\\begin{align}\n",
+    "    \\hat\\sigma_\\overline{X} = \\frac{\\hat\\sigma}{\\sqrt{N}}\\tag{2}\n",
+    "\\end{align}\n",
+    "$\n",
+    "\n",
+    "where $\\hat\\sigma_\\overline{X}$ is the estimated SEM, $\\hat\\sigma$ is the estimated standard deviation (see eq. 1) and $N$ is the number of samples. But what does it mean for samples to be uncorrelated?\n",
+    "\n",
+    "An example for uncorrelated samples would be the rolling of a dice. The outcome of each trial is completely independent to the previous trials. We might guess any number from 1 to 6, regardless of what has been the last result. The same could be true if we ran an experiment many times independently from one another and measured a quantity each time. By looking at one experimental value, we would'nt be able to predict the next one. The best guess would be simply the mean value of the entire series.\n",
```

Comment:
Consider:
```suggestion
    "An example for uncorrelated samples would be the rolling of a dice. The outcome of each trial is completely independent to the previous trials. We might guess any number from 1 to 6, regardless of what has been the last result. The same could be true if we ran an experiment many times independently from one another and measured a quantity each time. By looking at one experimental value, we would'nt be able to predict the next one. The best guess would be simply the mean value of the entire series. In the case of rolling a dice, correlations could for example be observed if it was more probable to obtain the same result as in the previous dice roll rather than another result.\n",
```